Engine_Revit_UI/Convert/Architecture/FromRevit/Ceilings.cs

```diff
@@ -76,6 +76,8 @@ public static List<oM.Architecture.Elements.Ceiling> CeilingsFromRevit(this Ceil
                 newCeiling.SetCustomData(ceiling);
 
                 newCeiling.UpdateValues(settings, ceiling);
+
+                newCeiling.CustomData.Add("CeilingPattern", ceiling.CeilingPattern(settings, planarSurface));
```

Comment:
Could we move the hardcoded string to some more central location? Maybe you could create a separate section in _Revit_Engine/Convert/Aliases.cs_?

It will be easier to manage migration in the future.